src/components/TagsFilter/TagsFilter.js

```diff
@@ -11,15 +11,25 @@ import {
 
 import CheckboxFacet from '../CheckboxFacet';
 
-const FILTER_NAME = 'tags';
+function TagsFilter(props) {
```

Comment:
Please use the arrow function for functional components.
And you can do the props destructuring in the function argumens

Reply:
Thanks, fixed